test/node/number_test.js

```diff
@@ -0,0 +1,149 @@
+"use strict";
+
+const expect = require("chai").expect;
+
+const BSON = require("../..");
+
+const BSON_DOUBLE_TYPE = 0x01;
+const BSON_INT32_TYPE = 0x10;
+
+const INT32_MAX = 2147483647;
+const INT32_MIN = -2147483648;
+
+const nanPayloadBuffer = Buffer.from("120000000000F87F", "hex");
+const nanPayloadDV = new DataView(
+  nanPayloadBuffer.buffer,
+  nanPayloadBuffer.byteOffset,
+  nanPayloadBuffer.byteLength
+);
+const nanPayloadDouble = nanPayloadDV.getFloat64(0, true);
+
+describe("serializing javascript numbers", () => {
+  it("should serialize a number that exceeds int32.max as a double", () => {
```

Comment:
Can we change the description language in these tests not to use "should" as in our testing guidelines?

Reply:
updated!